src/components/CallView/shared/VideoBottomBar.vue

```diff
@@ -20,87 +21,76 @@
 -->
 
 <template>
-	<div class="wrapper"
-		:class="{'wrapper--big': isBig}">
+	<div class="wrapper" :class="{'wrapper--big': isBig}">
 		<transition name="fade">
-			<div v-if="!connectionStateFailedNoRestart && model.attributes.raisedHand.state"
-				class="bottom-bar__statusIndicator">
-				<HandBackLeft class="handIndicator"
-					:size="18"
-					fill-color="#ffffff" />
+			<div v-if="showRaiseHandIndicator" class="status-indicator">
+				<HandBackLeft :size="18" fill-color="#ffffff" />
 			</div>
 		</transition>
-		<div v-if="!isSidebar"
-			class="bottom-bar"
-			:class="{'bottom-bar--video-on' : hasShadow, 'bottom-bar--big': isBig }">
+
+		<div v-if="!isSidebar" class="bottom-bar">
 			<transition name="fade">
-				<div v-show="showNameIndicator"
-					class="bottom-bar__nameIndicator"
-					:class="{'bottom-bar__nameIndicator--promoted': boldenNameIndicator}">
+				<div v-show="showParticipantName"
+					class="participant-name"
+					:class="participantNameClass">
 					{{ participantName }}
 				</div>
 			</transition>
```

Comment:
Not sure about this one. Name is separated from other media indicators, and visible on each `mouseover` event inside callView because of `showVideoOverlay`
So i would go with v-show here and v-if in other places

Reply:
If it is mouseover effect, maybe show it with CSS?

Reply:
Not exactly. The timer is defined by Grid.vue, and `showVideoOverlay` changed by that timer function passed as prop 
Grid.vue => Video.vue => VideoBottomBar.vue.
- If we want to test it properly, it requires some integration tests - do we need it here?
- If we want to change that behavior (with CSS), it should be done in Grid.vue - could it break current behavior? Now everything _(visually)_ works as before
- If it triggered quite often during the call - is it better to leave `v-show`? as i know, it just add `display: none` to the DOM element, without mounting/removing it, so it shoud be better for performance?